From 0b5ba3c05d423f22154a62e30aa75b7e33034a21 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 22 Sep 2020 13:32:04 +0200
Subject: IWYU: uint32_t requires stdint.h include in printing PageRange.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix this error:
../../printing/page_range.h:20:3: error: ‘uint32_t’ does not name a type

Introduced in this commit:
https://chromium-review.googlesource.com/c/chromium/src/+/2351580

Bug: 951519
Change-Id: I0c86cfc5dc711b062021dd8661d27d6442e7d47d
---
 printing/page_range.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/printing/page_range.h b/printing/page_range.h
index 0d262a69a872..20e84faef960 100644
--- a/printing/page_range.h
+++ b/printing/page_range.h
@@ -5,6 +5,8 @@
 #ifndef PRINTING_PAGE_RANGE_H_
 #define PRINTING_PAGE_RANGE_H_
 
+#include <stdint.h>
+
 #include <vector>
 
 #include "printing/printing_export.h"
-- 
2.25.1

